vortex-buffer/src/buffer_mut.rs

```diff
@@ -526,8 +525,8 @@ impl<T> BufferMut<T> {
                 dst = dst.add(1);
             }
         });
-        // TODO(joe): replace with ptr_sub when stable
-        let length = self.len() + unsafe { dst.byte_offset_from(begin) as usize / size_of::<T>() };
+
+        let length = self.len() + unsafe { dst.offset_from_unsigned(begin) };
```

Comment:
the point of the todo was to avoid the two ops, which this doesn't do but I guess its a pretty small win

Reply:
So what is the thing you want to do here? Do the sub from the head of the bytes?